src/View/Window.vala

```diff
@@ -342,6 +342,11 @@ namespace Marlin.View {
                 });
             });
 
+            tabs.tab_added.connect ((tab) => {
```

Comment:
Can you clarify why this was not present in the code before and why it is needed now, and what it actually does?